src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1281,6 +1281,28 @@ public void Start_HasStandardErrorEncodingNonRedirected_ThrowsInvalidOperationEx
             Assert.Throws<InvalidOperationException>(() => process.Start());
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [Fact]
+        public void Start_RedirectStandardOutput_StartAgainWithoutRedirect()
+        {
+            var process = new Process
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = "cmd",
+                    Arguments = "/C dir",
+                    RedirectStandardOutput = true,
+                }
+            };
+
+            process.Start();
+            process.BeginOutputReadLine();
+
+            process.Start();
```

Comment:
Are there things we can assert here, such as the first call returning true and the second returning false? (I've not validated that's actually the case, just speculating.)